### PR TITLE
start to clean setup.py

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -221,10 +201,10 @@
                 extras_require[new_key] = extras_require.pop(key)
 
 everything = set()
-for key, deps in extras_require.items():
+for key, deps in sorted(extras_require.items()):
     if ':' not in key:
         everything.update(deps)
-extras_require['all'] = everything
+extras_require['all'] = list(sorted(everything))
```
Comment (terrdavis) [inline setup.py:207]: sorted already returns a list!

Comment (Carreau) [reply setup.py:207]: Ih, I'm not sure hy I shoudl it returned a genrator.

Comment (Carreau) [reply setup.py:207]: Damn I can't type today.

**Comment by Carreau** [comment]
@meeseeksdev suggestions

**Comment by Carreau** [comment]
(sorry this was a test with  bot to try to automatically suggest back reformating of changed lines)>

**Pull request closed without merging**